docs/source/55_api.md

```diff
@@ -339,6 +342,134 @@ val bcastFeatures = sc.loadFeatures("my/features.adam").broadcast()
 val readsByFeature = reads.broadcastRegionJoinAgainst(bcastFeatures)
 ```
 
+To demonstrate how the RegionJoin APIs can be used to answer scientific
+questions, we will walk through three common queries that can be written using
+the RegionJoin API. First, we will perform a simple filter on genotypes based
+on a file of features. We will then demonstrate a join and group by on variants
+and features, providing variant data grouped by the feature they overlap.
+Finally, we will separate reads into those that overlap and those that do not
+overlap features from a feature file.
+
+Each of these demonstrations illustrates the difference between calling the
+ShuffleRegionJoin and BroadcastRegionJoin and provides example code that can
+be expanded from.
+
+###### Filter Genotypes by Features
+
+This query joins an RDD of Genotypes against an RDD of Features using an inner
+join. The inner join will result in an RDD of key-value pairs, where the key is
+a genotype and the value is the feature that it overlaps. Because this is an
+inner join, records from either dataset that don't pair to the other are
+automatically dropped, providing the filter we are interested in. This query is
+useful for trying to identify genotypes that overlap features of interest. For
+example, if our feature file contains all the exonic regions of the genome,
+this query would extract all genotypes that fall in exonic regions.
+
+```scala
+// Inner join will filter out genotypic data not represented in the feature dataset
+val genotypes = sc.loadGenotypes(“my/genotypes.adam”)
+val features = sc.loadFeatures(“my/features.adam”)
+
+// We can use ShuffleRegionJoin…
+val joinedGenotypesShuffle = genotypes.shuffleRegionJoin(features)
+
+// …or BroadcastRegionJoin
+val joinedGenotypesBcast = features.broadcastRegionJoin(genotypes)
+
+// In the case that we only want Genotypes, we can use a simple predicate
+val filteredGenotypesShuffle = joinedGenotypesShuffle.rdd.map(_._1)
+
+val filteredGenotypesBcast = joinedGenotypesBcast.rdd.map(_._2)
+```
+
+After the join, we can perform a predicate function on the resulting RDD to
```

Comment:
This is unnecessarily verbose. Suggest:

> Since we are interested in the `Genotype`s that overlap a feature, we map over the tuples and select just the `Genotype`.